src/extra/Markdown/index.tsx

```diff
@@ -1,743 +1,81 @@
 import * as React from 'react';
-import marked from 'marked';
-import highlightJS from 'highlight.js';
```

Comment:
Do we still get language highlighting with the new setup? I see there is `rehype-highlight` but it is significantly smaller compared to highlightJS, so I'm just wondering what we're losing and if that is important or not.

Reply:
Yep - seconded that we want to make sure we don't lose highlighting support. This is very useful in Jellyfish.

(If it turns out highlightJS isn't needed here it should be removed as a dependency entirely (package.json) as this is the only place it's referenced).

Reply:
`rehype-highlight` uses [`lowlight`](https://github.com/wooorm/lowlight) which support everything supported by `highlight.js`, works only for vdom libs like react, preact, etc. so we only lose weight 🏋️‍♀️